assets/js/googlesitekit/notifications/components/layout/SingleColumnNotificationWithSVG.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { Cell, Grid, Row } from '../../../../material-components';
+
+export default function SingleColumnNotificationWithSVG( {
+	id,
+	title,
+	description,
+	actions,
+	SVG,
+} ) {
+	return (
+		<div className="googlesitekit-widget-context">
+			<Grid className="googlesitekit-widget-area">
```

Comment:
This looks quite hackish – why are doing this especially just for a single notification?

Edit: I see now that this is done in other notification components as well, which I don't understand. We shouldn't emulating this in new notifications, but maybe it's only intended to be temporary? If not, let's plan on refactoring this out in a future issue.

Reply:
This started off when we "copy-pasted" the first setup CTA (I think Audience Segmentation or Consent Mode) and this has been copy-pasted since in all Setup Banners that are "separate" like widgets and not "attached" to the Navigation bar like our other Banner Notifications. We made a conscious decision not to spend too much time on restyling everything as this was part of a future phase and not in scope for this epic.

This will definitely all be restyled in BNR Phase 3 - I will be adding these notes to the Design Doc and future issues.